src/components/common/input/Input.jsx

```diff
@@ -1,8 +1,11 @@
 import { useState } from 'react';
+import classNames from 'classnames/bind';
 import styles from './Input.module.css';
 import PropTypes from 'prop-types';
 
-export const Input = ({ onChange, ...otherProps }) => {
+const cx = classNames.bind(styles);
+
+export const Input = ({ onChange, errorMsg, ...otherProps }) => {
```

Comment:
Input should have a type

Reply:
As far as I remember, none of the previous tasks involved using TypeScript. So I can't add types for regular functions. Do you want me to add TypeScript to the project in this task?

Reply:
I mean native input should have a type

Reply:
it's passed in 'otherProps'

Reply:
done